packages/react-components/react-menu/src/components/MenuItem/useMenuItemStyles.styles.ts

```diff
@@ -37,10 +37,13 @@ const useStyles = makeStyles({
     backgroundColor: tokens.colorNeutralBackground1,
     paddingRight: '6px',
     paddingLeft: '6px',
-    height: '32px',
+    boxSizing: 'border-box',
+    maxWidth: '290px',
     minHeight: '32px',
+    flexShrink: 0,
     display: 'flex',
-    alignItems: 'center',
+    alignItems: 'start',
```

Comment:
why `start`? this will top align all content

Reply:
synced on call, we do need it to make menu content, icon top aligned 😭

Reply:
Let's keep this thread open to prevent the PR from being merged until the latest upgrade goes in